src/main/java/com/github/jmchilton/blend4j/galaxy/HistoriesClient.java

```diff
@@ -37,6 +38,20 @@
   HistoryExport exportHistory(String historyId);
   
   /**
+   * Deletes the given History from Galaxy (this will not purge).
+   * @param historyId  The id of the History to delete.
+   * @return A {@link ClientResponse} for this request.
+   */
+  ClientResponse deleteHistoryResponse(String historyId);
```

Comment:
A `ClientResponse` encodes the HTTP status code along with other information about the response body.  The response body, if in JSON format, can be converted to a Java object with `ClientResponse.getEntity(HistoryDeleteResponse.class)`, assuming the JSON matches the fields in the object.  The response I get from a history after deleting is:

```json
{
    "deleted": true, 
    "id": "b79aa7e43ec1b406"
}
```

Which was pretty simple, so I created a new object to encapsulate it.

The response I get from deleting a library is:

```json
{
    "can_user_add": true, 
    "can_user_manage": true, 
    "can_user_modify": true, 
    "deleted": true, 
    "description": "", 
    "id": "42a2c611109e5ed3", 
    "model_class": "Library", 
    "name": "test-delete-library86497a37-f9e7-4ee8-9c2f-4df3070d4c67", 
    "public": true, 
    "root_folder_id": "F9752b387803d3e1e", 
    "synopsis": ""
}
```

This is more complicated so I didn't build a response object for it.